.cp2k/Makefile

```diff
@@ -183,24 +180,24 @@ LIBSMM_ACC_ABS_DIR := $(shell find $(SRCDIR) -type d -name "libsmm_acc")
 
 ALL_PKG_FILES := $(shell find $(SRCDIR) -name "PACKAGE")
 OBJ_SRC_FILES  = $(shell cd $(SRCDIR); find . ! -name "dbcsr_api_c.F" ! -name "dbcsr_tensor_api_c.F" -name "*.F")
-OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -name "*.c")
+OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -path ../src/acc/opencl -prune -type f -name "*.c")
```

Comment:
I have some issues to build with OpenCL now, but do not worry. I will send my fixes on top of yours. I think pruning is not necessary since the related translation units are simply empty if `__OPENCL` is not defined.

Reply:
In fact, your introduction of `USE_ACCEL` for the Makefile is sensible and in sync with what's introduced with CMake. I will need to fix-up with OpenCL backend in any case simply to match `USE_ACCEL` with `opencl` rather than falling into the "else-case". Further, OpenCL BE can now pickup own parameter files, which is simply not implemented yet in the Makefile.